File: 1603-RunningSumOf1dArray/1603-RunningSumOf1dArray.java
// Last updated: 9/11/2025, 12:31:05 PM
class Solution {
    public int[] runningSum(int[] nums) {
        for(int i =1; i<nums.length;i++){
                nums[i]+=nums[i-1]; 
        }
        return nums;
    }
}

File: 1603-RunningSumOf1dArray/1603-RunningSumOf1dArray.py
# Last updated: 9/11/2025, 12:31:02 PM
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        ans = [nums[0]] + [0] * (len(nums)-1)
        for i in range(1, len(nums)):
            ans[i] = ans[i-1] + nums[i]
        return ans